### ISSUE TITLE
Mark var folder as excluded when symfony plugin is enabled

### ISSUE THREAD

**Issue by alexander-schranz** [Haehnchen/idea-php-symfony2-plugin#1387]
If you open a symfony project here is mostly a `var` folder in the project directory. Which includes fetch and other variable files which changes frequently and making the ide slower. Would be awesome if the plugin by default would set the `var` directory as excluded.

**Comment by ruudk** [comment]
Why should this be a responsibility of this plugin? Even though Symfony uses this directory structure, it does not mean that the contents of `var` are only for Symfony. I can also store stuff there.

**Comment by alexander-schranz** [comment]
@ruudk yes I store also other files into the `var` folder but as the name of the folder says they are variable and so they don't need to be indexed by the IDE as they will not be referenced in my code. But it just a default so if you want to index it you could then add your folder to be indexed.

**Comment by mareg** [comment]
I think it would be neat to exclude by default jus `var/cache`.
